charts/prometheus/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: prometheus
-appVersion: v2.44.0
-version: 22.6.6
+appVersion: v2.45.0
+version: 22.7.0
```

Comment:
@arukiidou 
Thanks for the contribution. Patch version increasing should work. If minor version increasing is required, please write up what the user should know on upgrading procedure and / or what features drop / change in prometheus.

Reply:
@zanhsieh 
- I think that's the case with the major version.
- No specific procedure is required
  - `helm upgrade prom ./ -n kube-system`, it works on.
  - No feature drop or significant changes [in the release notes](https://github.com/prometheus/prometheus/releases/tag/v2.45.0)

Reply:
I did it.